interactions/api/models/misc.py

```diff
@@ -203,7 +203,7 @@ class Format:
     def stylize(cls, format: str, **kwargs) -> str:
         r"""
         This takes a format style from the object and
-        converts it into a useable string for ease.
+        converts it into an useable string for ease.
```

Comment:
`a usable`
Useable *is* correct, but usable is used more, in both American and British English.